### PR TITLE
timing: get current k8s env

### PR DESCRIPTION
**Pull request by maiamcc**
not currently super necessary but we'll probs want this when outputting to csv etc.

### CODE REVIEW

File: scripts/timing.py [hunk 1]
```diff
@@ -149,6 +156,26 @@ def call_or_error(cmd: List[str]):
         raise Exception('Command {} exited with exit code {}'.format(cmd, return_code))
 
 
+def get_k8s_env() -> K8sEnv:
+    """Get current Kubernetes env. (or throw an exception)."""
+    p = subprocess.Popen(['kubectl', 'config', 'view', '-o', 'template', '--template={{ index . "current-context" }}'], stdout=subprocess.PIPE)
```
Comment (nicks) [inline scripts/timing.py]: are you aware of 'kubectl config current-context'?

this seems...overly complicated

Comment (nicks) [inline scripts/timing.py]: did you consider using subprocess.check_output? this is a very common pattern and it seems weird to reach for these lower-level primitives.

Comment (maiamcc) [reply scripts/timing.py]: ...i did NOT consider that, no. i was lead astray by a stack overflow article 😳

Comment (maiamcc) [reply scripts/timing.py]: nnnnno. again, stack overflow strugglebus. 😳

**Pull request merged**